# HG changeset patch
# User Rik <rik@octave.org>
# Date 1551074524 28800
#      Sun Feb 24 22:02:04 2019 -0800
# Branch stable
# Node ID 1ff470c9e9c2edeb8f70614fdf7f2f6ee07e9a64
# Parent  bb3a441addb0204ed2fcf4e696cebeb5dff167e0
Mark strings for translation in variable editor (bug #55772).

* variable-editor.cc (variable_editor_view::createColumnMenu,
variable_editor_view::createRowMenu): Use tr() wrapper around static strings
rather than code which evaluates to string.

diff --git a/libgui/src/variable-editor.cc b/libgui/src/variable-editor.cc
--- a/libgui/src/variable-editor.cc
+++ b/libgui/src/variable-editor.cc
@@ -663,17 +663,17 @@ namespace octave
 
     if (! whole_columns_selected || ! current_column_selected)
       {
         selectColumn (index);
         column_selection_count = 1;
       }
 
     QString column_string
-      = tr (column_selection_count > 1 ? " columns" : " column");
+      = column_selection_count > 1 ? tr (" columns") : tr (" column");
 
     QMenu *menu = new QMenu (this);
 
     add_edit_actions (menu, column_string);
 
     menu->addSeparator ();
 
     QSignalMapper *plot_mapper = make_plot_mapper (menu);
@@ -710,17 +710,17 @@ namespace octave
     int rowselection_count = nothingSelected ? 0 : (coords[3] - coords[2] + 1);
 
     if (! whole_rows_selected || ! current_row_selected)
       {
         selectRow (index);
         rowselection_count = 1;
       }
 
-    QString row_string = tr (rowselection_count > 1 ? " rows" : " row");
+    QString row_string = rowselection_count > 1 ? tr (" rows") : tr (" row");
 
     QMenu *menu = new QMenu (this);
 
     add_edit_actions (menu, row_string);
 
     menu->addSeparator ();
 
     QSignalMapper *plot_mapper = make_plot_mapper (menu);
